integrations/standalone/vite.config.ts

```diff
@@ -7,7 +7,7 @@ export default defineConfig({
   build: {
     outDir: 'build',
     chunkSizeWarningLimit: 5000,
-    rollupOptions: { input: { index: './index.html' } }
+    rollupOptions: { input: { index: './index.html', mock: './mock.html'  } }
```

Comment:
pls filter the mock output from the web jar. e.g. like https://github.com/axonivy/form-editor-client/blob/master/integrations/standalone/assembly.xml#L18-L19